.github/actions/cleanup/pe.go

```diff
@@ -0,0 +1,78 @@
+package main
+
+import (
+	"context"
+	"fmt"
+
+	"go.mongodb.org/atlas/mongodbatlas"
+)
+
+func deleteAllPE(ctx context.Context, client mongodbatlas.PrivateEndpointsService, projectID string) error {
```

Comment:
I wonder why functions for Private Endpoints are not exported while for network peers are exported? What's the difference?

Reply:
What kind of functions do you mean? DeleteAllPrivateEndpoint function from atlas operator:
```
func DeleteAllPrivateEndpoints(ctx *workflow.Context, projectID string) workflow.Result {
	atlasPEs, err := getAllPrivateEndpoints(ctx.Client, projectID)
	if err != nil {
		return workflow.Terminate(workflow.Internal, err.Error())
	}

	endpointsToDelete := getEndpointsNotInSpec([]mdbv1.PrivateEndpoint{}, atlasPEs)
	return deletePrivateEndpointsFromAtlas(ctx, projectID, endpointsToDelete)
}
```
I cant use it for cleaner

Reply:
Well, ofc, not as is, I mean like it is possible to change these functions so you can reuse them. My point was not to have the same basic logic in two places but it's fine as is.